src/model/UserModel.js

```diff
@@ -0,0 +1,48 @@
+import {
+  PRICE_RANGE,
+  NUMBER_RANGE,
+  LOTTO_NUMBER_LENGTH,
+} from '../constant/system.js';
+import Lotto from '../Lotto.js';
+import { Random } from '@woowacourse/mission-utils';
+
+export default class UserModel {
+  #price;
+  #lottos;
+
+  constructor(price) {
+    this.#price = price;
+    this.#lottos = this.#createLottos();
+  }
+
+  #createLottos() {
+    const lottos = [];
+    const lottoLength = this.#price / PRICE_RANGE.MIN;
+
+    for (let i = 0; i < lottoLength; i += 1) {
+      const sortedRandomNumber = Random.pickUniqueNumbersInRange(
+        NUMBER_RANGE.MIN,
+        NUMBER_RANGE.MAX,
+        LOTTO_NUMBER_LENGTH,
+      ).sort((a, b) => a - b);
+      lottos.push(new Lotto(sortedRandomNumber));
+    }
+
+    return lottos;
+  }
```

Comment:
`Array.from`을 사용할 수 있을 것 같아요!
반복문과 `push` 로직을 없앨 수 있답니다 👍 

```suggestion
  #createLottos() {
    const lottoLength = this.#price / PRICE_RANGE.MIN;

    const lottos = Array.from({ length: lottoLength }, () => {
      const sortedRandomNumber = Random.pickUniqueNumbersInRange(
        NUMBER_RANGE.MIN,
        NUMBER_RANGE.MAX,
        LOTTO_NUMBER_LENGTH,
      ).sort((a, b) => a - b);
      return new Lotto(sortedRandomNumber);
    })
  
    return lottos;
  }
```

Reply:
자바스크립트 내장 메서드를 사용하여 더 간단히 표현하는 방법이 있군요..!

좋은 피드백 감사합니당